src/datasets/builder.py

```diff
@@ -709,7 +721,7 @@ def download_and_prepare(
                     force_download=bool(download_mode == DownloadMode.FORCE_REDOWNLOAD),
                     force_extract=bool(download_mode == DownloadMode.FORCE_REDOWNLOAD),
                     use_etag=False,
-                    use_auth_token=use_auth_token,
+                    use_auth_token=self.use_auth_token,
```

Comment:
And following the suggested change of my previous comment, we can keep it as it was:
```suggestion
                    use_auth_token=use_auth_token,
```